workspace/src/string-transform.js

```diff
@@ -53,25 +247,24 @@ export function transform(op0, op1) {
   if (op0.type === "ins") {
```

Comment:
you could potentially use a dictionary to realize this function. @jiamingguo @jam-world 

Use 
```
transforms = {'ins,del': transform_ins_del, ...}
```